lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -349,21 +339,26 @@ def is_privilege_single_policy_class(user_or_attribute, operation, object_or_att
       end
     end
 
+
     def is_privilege_multiple_policy_classes(user_or_attribute, operation, object_or_attribute, policy_classes_containing_object)
-      base_scope =  if operation.is_a?(class_for_type('operation'))
-        associations_between(user_or_attribute, object_or_attribute).where(id: operation.policy_element_associations)
-      else
-        associations_between(user_or_attribute, object_or_attribute).joins(:operations).where(policy_elements: {unique_identifier: operation})
-      end
+      #Outstanding active record sql adapter prevents chaining an additional where using the association.
+      # TODO: fix when active record is fixed
       policy_classes_containing_object.all? do |pc|
-        base_scope.where(object_attribute_id: pc.ancestors).any?
+        if operation.is_a?(class_for_type('operation'))
+          associations_between(user_or_attribute, object_or_attribute).where(id: operation.policy_element_associations.to_a, object_attribute_id: pc.ancestors).any?
```

Comment:
`operation.policy_element_associations.to_a` looks like a performance sadface, is it possible to keep this to a single SQL query?